src/controllers/audience/audienceController.ts

```diff
@@ -238,8 +274,12 @@ export class AudienceController {
 
 			if (!errors.size) {
 				res.locals.audience.requiredBy = date.toDate()
-				await this.learningCatalogue.updateCourse(res.locals.course)
-				res.redirect(`/content-management/courses/${req.params.courseId}/audiences/${req.params.audienceId}/configure`)
+				await this.learningCatalogue
+					.updateCourse(res.locals.course)
```

Comment:
Shouldn't this be done with updateAudience, instead of updateCourse, like in setAreasOfWork()?